dao/src/main/java/org/thingsboard/server/dao/sql/attributes/AttributeKvRepository.java

```diff
@@ -59,5 +59,7 @@ List<String> findAllKeysByDeviceProfileId(@Param("tenantId") UUID tenantId,
     @Query(value = "SELECT DISTINCT attribute_key FROM attribute_kv WHERE entity_type = :entityType " +
             "AND entity_id in :entityIds ORDER BY attribute_key", nativeQuery = true)
     List<String> findAllKeysByEntityIds(@Param("entityType") String entityType, @Param("entityIds") List<UUID> entityIds);
-}
 
+    @Query(value = "SELECT * FROM attribute_kv WHERE attribute_key = :attributeKey AND entity_id in :entityIds ", nativeQuery = true)
+    List<AttributeKvEntity> findAllValuesByEntityIds(@Param("attributeKey") String attributeKey, @Param("entityIds") List<UUID> entityIds);
```

Comment:
What is the reason for 'SELECT *' and nativeQuery? 
JPQL is more preferred to see issue during compilation, and not runtime.
Is it possible to use next query ? 
**@Query(value = "SELECT a FROM AttributeKvEntity a WHERE a.id.attributeKey = :attributeKey " +
            "AND a.id.entityId IN :entityIds ")**